src/index.js

```diff
@@ -1,29 +1,22 @@
 import './style.css';
```

Comment:
- [ ] I think there is an issue with the To-Do list style because the icons are being shown outside of the container. I would suggest fixing this issue in order to match the template :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/145219162-da857534-263c-4c25-aad5-45c38e940575.png)
**Note: Items has been added manually**